### PR TITLE
Fix README link

### PR DESCRIPTION
**Pull request by prateekpandey14**
There is broken link which points to aggregated-api-servers.md file in kubernetes/community repository.
@deads2k

### CODE REVIEW

**Comment by nikhita** [comment]
@prateekpandey14 Similar to my comment here (https://github.com/kubernetes/apiextensions-apiserver/pull/14#issuecomment-331970140), can you fix the typo in the passing?

**Comment by prateekpandey14** [comment]
Done. Thanks @nikhita

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # kube-aggregator
 ## Coming Soon!
 
-Implements https://github.com/kubernetes/community/blob/master/contributors/design-proposals/aggregated-api-servers.md.
+Implements : https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/aggregated-api-servers.md.
```
Comment (caesarxuchao) [inline README.md]: What's the extra ": " for?

Comment (prateekpandey14) [reply README.md]: removed. @caesarxuchao Thanks

Comment (prateekpandey14) [reply README.md]: removed extra ":", Thanks @caesarxuchao

**Comment by prateekpandey14** [comment]
@caesarxuchao Can we merge this now.

**Comment by caesarxuchao** [comment]
Thanks.

@sttts can we manual merge README changes?

**Comment by sttts** [comment]
@caesarxuchao if the files are not in the staging folder, yes.

**Comment by caesarxuchao** [comment]
Thanks for the confirmation. README is not in the staging folder: https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/kube-aggregator.

Merging.

**Pull request merged**